aten/src/ATen/cpu/sleef/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+IF(MSVC)
+  option(BUILD_SHARED_LIBS "Build shared libs" ON)
+ELSE(MSVC)
+  option(BUILD_SHARED_LIBS "Build shared libs" OFF)
+ENDIF(MSVC)
+option(SLEEF_SHOW_ERROR_LOG "Show cmake error log." OFF)
+
+set(SLEEF_VERSION_MAJOR 3)
+set(SLEEF_VERSION_MINOR 2)
```

Comment:
Picture in my head: Someone's gonna upgrade the sleef submodule and forget to update these version numbers. ;)

Reply:
Yes :(